utilities/build_envs.sh

```diff
@@ -54,20 +54,30 @@ case "${ARCH?}" in
         )
         ;;
     aarch64)
-        JULIA_CPU_TARGETS+=(
-            # Absolute base aarch64 feature set
-            "generic"
-            # Cortex A57, Example: NVIDIA Jetson TX1, Jetson Nano
-            "cortex-a57"
-            # Cavium ThunderX2T99, a common server architecture
-            "thunderx2t99"
-            # NVidia Carmel, e.g. Jetson AGX Xavier; serves as a baseline for later architectures
-            "carmel,clone_all"
-            # Apple M1
-            "apple-m1,base(3)"
-            # Vector-length-agnostic common denominator between Neoverse V1 and V2, recent Arm server architectures
-            "neoverse-512tvb,base(3)"
-        )
+        case "${OS?}" in
+            macos)
+                JULIA_CPU_TARGETS+=(
+                    # Absolute base aarch64 feature set
+                    "generic"
+                    # Apple M1
+                    "apple-m1,clone_all"
+                )
+                ;;
+            *)
+                JULIA_CPU_TARGETS+=(
+                    # Absolute base aarch64 feature set
+                    "generic"
+                    # Cortex A57, Example: NVIDIA Jetson TX1, Jetson Nano
+                    "cortex-a57"
+                    # Cavium ThunderX2T99, a common server architecture
+                    "thunderx2t99"
+                    # NVidia Carmel, e.g. Jetson AGX Xavier; serves as a baseline for later architectures
+                    "carmel,clone_all"
+                    # Vector-length-agnostic common denominator between Neoverse V1 and V2, recent Arm server architectures
```

Comment:
Why not apple-m1?

Reply:
ah-sorry (asahi), I forgot about using other OSs on macs

Reply:
I haven't checked carefully, but it may also be usable by Neoverse (since the `neoverse-512tvb` target isn't doing anything right now)